src/Features/CSharp/Portable/Snippets/CSharpConstructorSnippetProvider.cs

```diff
@@ -89,7 +89,7 @@ private static string GetIndentation(Document document, SyntaxNode node, SyntaxF
         protected override async Task<Document> AddIndentationToDocumentAsync(Document document, int position, ISyntaxFacts syntaxFacts, CancellationToken cancellationToken)
         {
             var root = await document.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
-            var snippet = root.GetAnnotatedNodes(_findSnippetAnnotation).FirstOrDefault();
+            var snippet = root.GetAnnotatedNodes(_findSnippetAnnotation).First();
```

Comment:
FYI @akhera99 for snippet related change

Reply:
I figured it would given that, when we annotate the added snippet, we throw if the added snippet can't be found. I think it's safer to keep it as FirstOrDefault, but I believe we bail out earlier in case it's not found.